build.gradle

```diff
@@ -141,6 +144,16 @@ if (file('.git').exists()) {
   }
 }
 
+def shouldFilterFlakyTests = gradle.startParameter.taskNames.any{it.contains("FlakyRetry")}
+if (shouldFilterFlakyTests) {
+  logger.warn('''************************************************************************
+  a FlakyRetry task was included in task list ->
+  Failing tests will not cause a build failure. Test classes with failures will be run again to
```

Comment:
nit: Would saying "Failing tests will be retried once" be clearer?

Reply:
"Failing tests will be retried once in order to test whether they are flaky."